GANDLF/utils/gandlf_logger.py

```diff
@@ -23,13 +24,21 @@ def _flush_to_console():
     logging.root.addHandler(console_handler)
 
 
+def _create_tmp_log_file():
+    tmp_dir = Path(tempfile.gettempdir())
+    log_dir = Path.joinpath(tmp_dir, "gandlf")
+    log_dir.mkdir(parents=True, exist_ok=True)
+    log_file = Path.joinpath(log_dir, "gandlf.log")
```

Comment:
I think from a user standpoint, putting the logs in `${tmp_dir}/.gandlf/${timestamp}.log` makes more sense (and it is also a more standardized approach to dealing with user-level logs):

```suggestion
    log_dir = Path.joinpath(tmp_dir, ".gandlf")
    log_dir.mkdir(parents=True, exist_ok=True)
    log_file = Path.joinpath(log_dir, get_unique_timestamp() + ".log")
```